src/libraries/System.Drawing.Common/tests/System.Drawing.Common.Tests.csproj

```diff
@@ -3,7 +3,7 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <IncludeRemoteExecutor>true</IncludeRemoteExecutor>
     <TargetFrameworks>$(NetCoreAppCurrent)-windows;$(NetCoreAppCurrent)-Unix;net48</TargetFrameworks>
-    <IgnoreForCI Condition="'$(TargetOS)' == 'Browser'">true</IgnoreForCI>
+    <IgnoreForCI Condition="'$(TargetsMobile)' == 'true'">true</IgnoreForCI>
```

Comment:
Should we also make the local run clean? i.e, tweaking: https://github.com/dotnet/runtime/blob/dbb2b53e0caeeb32888f83a5c6fe6db8525398de/src/libraries/Common/tests/TestUtilities/System/PlatformDetection.cs#L93

We could tweak that to instead try and create a new bitmap inside a try catch... and then if it fails, return false?

Reply:
We can't easily add a reference to System.Drawing.Common in TestUtilities.  So, it'll just return false if running on mobile.